pkg/controller/machine/controller.go

```diff
@@ -154,6 +154,16 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{}, err
 		}
 
+		if m.Status.NodeRef != nil {
+			if m.Status.NodeRef != nil {
```

Comment:
Am I just reading this wrong? These nested if statements seem identical.

Reply:
thanks!